quantum/process_keycode/process_ucis.c

```diff
@@ -60,31 +60,12 @@ __attribute__((weak)) void qk_ucis_symbol_fallback(void) {
 
 __attribute__((weak)) void qk_ucis_cancel(void) {}
 
-void register_ucis(const char *hex) {
-    for (int i = 0; hex[i]; i++) {
-        uint8_t kc = 0;
-        char    c  = hex[i];
-
-        switch (c) {
-            case '0':
-                kc = KC_0;
-                break;
-            case '1' ... '9':
-                kc = c - '1' + KC_1;
-                break;
-            case 'a' ... 'f':
-                kc = c - 'a' + KC_A;
-                break;
-            case 'A' ... 'F':
-                kc = c - 'A' + KC_A;
-                break;
-        }
-
-        if (kc) {
-            register_code(kc);
-            unregister_code(kc);
-            wait_ms(UNICODE_TYPE_DELAY);
-        }
+void register_ucis(const uint32_t *codes) {
+    uint8_t input_mode = get_unicode_input_mode();
+    for(int i = 0; i < UCIS_MAX_CODE_LENGTH && codes[i]; i++) {
+        uint32_t code = codes[i];
+        register_unicode(code, input_mode);
+        wait_ms (UNICODE_TYPE_DELAY);
```

Comment:
Rename parameter and fix up whitespace.
```suggestion
void register_ucis(const uint32_t *code_points) {
    uint8_t input_mode = get_unicode_input_mode();
    for (int i = 0; i < UCIS_MAX_CODE_POINTS && code_points[i]; i++) {
        register_unicode(code_points[i], input_mode);
        wait_ms(UNICODE_TYPE_DELAY);
```